src/scandipwa/plugin/PayPal.plugin.js

```diff
@@ -0,0 +1,133 @@
+import PayPal from '../app/component/PayPal';
+
+export const PAYPAL_EXPRESS_CREDIT = 'paypal_express_bml';
+export const PAYPAL_EXPRESS = 'paypal_express';
+export class PayPalPlugin {
```

Comment:
And I am also thinking. This is not extensible class. Should it be? We can do the dev extension like this - but I am thinking if plugin must be extensible ???